### PR TITLE
portfolio cards

### PR DESCRIPTION
**Pull request by renercrows**
### Portfolio Cards

- Added cards with a grid.
- Added some about me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,5 +40,157 @@ <h2>I'm a software developer</h2>
...
+            <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtittle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
```
Comment (Kakalanp) [inline index.html:58]: Your *Used languages* section is looking pretty good, but kindly add a tiny bit of `border-radius` to them, also, try fixing this issue, where the items get outside the card :mag: 

image

File: style.css [hunk 2]
```diff
@@ -118,3 +105,523 @@ h2 {
...
+
+.blank2 {
+  display: flex;
+  background-color: #ebf0ee;
+  height: 352px;
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 10px;
+  margin-bottom: -50px;
+}
```
Comment (Kakalanp) [inline style.css:361]: Try adding some `border-radius` to these items too, so they look more alike the Figma design :nerd_face: 

image

File: style.css [hunk 3]
```diff
@@ -118,3 +105,523 @@ h2 {
...
+
+.cardtittle2 {
+  display: flex;
+  align-items: center;
+  font-family: Inter, sans-serif;
+  padding-left: 90px;
+  margin-top: 120px;
+  margin-bottom: -50px;
+  color: #344563;
+}
```
Comment (Kakalanp) [inline style.css:383]: Looks like your **font** is missing a little **weight** here :thinking: Maybe try adding some `font-weight` to it, to make it look closer to the Figma design :rocket: 

image

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @renercrows ! hi

### Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @renercrows,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 




Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**